_posts/When you’re done naming the file, add the following content to your file:  --- layout: slide title: "Welcome to our second slide!" --- Your text Use 0000-01-02-jeannealli.md

```diff
@@ -0,0 +1,7 @@
+When you’re done naming the file, add the following content to your file: 
```

Comment:
--- layout: slide title: "Welcome to our second slide!" --- Your text Use 0000-01-02-jeannealli.md